hardware/setup/snap_build.tcl

```diff
@@ -173,16 +206,25 @@ append IMAGE_NAME [format {_%s_%s_%s} $RAM_TYPE $fpgacard $TIMING_TNS]
 set step     write_bitstream
 set logfile  $log_dir/${step}.log
 set command  "write_bitstream -force -file ./Images/$IMAGE_NAME"  
-puts [format "%-*s %-*s %-*s %-*s"  $widthCol1 "" $widthCol2 "generating bitstreams" $widthCol3 "" $widthCol4 "[clock format [clock seconds] -format %H:%M:%S]"]
+
```

Comment:
Missing image name change for factory image. 
Please add to line 203: append IMAGE_NAME [expr {$factory_image == "TRUE" ? "_FACTORY" : ""}]